Objects/Converters/ConverterAutocadCivil/ConverterAutocadCivilShared/ConverterAutocadCivil.Civil.cs

```diff
@@ -1142,6 +1052,7 @@ public Pipe PipeToSpeckle(CivilDB.Pipe pipe)
     // assign additional pipe props
     AddNameAndDescriptionProperty(pipe.Name, pipe.Description, specklePipe);
 
+    // TODO: use !IsFatal() here
```

Comment:
ouch, what is throwing here? Is it the ToString() or can some things be null?

Reply:
Accessing some properties on the `pipe` object may throw for undocumented reasons (usually if the property doesn't exist or isn't applicable). It's worth trying to add as many as possible to the pipe, not knowing which may throw, thus the individual try catches